service/Dockerfile

```diff
@@ -25,4 +27,5 @@ RUN cargo build --release -p service --bin post-service
 FROM debian:bookworm-slim AS runtime
 WORKDIR /service
 COPY --from=builder /service/target/release/post-service /usr/local/bin
```

Comment:
IMO these packets should be added there, into the final image.

Reply:
Ah yeah, true, I had them there and then moved to "consolidate" :D